api/apps/api/src/modules/geo-features/geo-feature.api.entity.ts

```diff
@@ -56,6 +57,9 @@ export class GeoFeature {
   @Column('varchar')
   tag!: FeatureTag;
 
+  @Column('varchar', { name: 'creation_status' })
+  creationStatus?: JobStatus;
```

Comment:
Why is this field necessary? Actual status usually come from ApiEvents (and in this case, should rather be related to some Scenario* event)

Reply:
I think this was done to keep to the current table structure, but we could probably even drop this column and related entity property, especially if doing the insert synchronously as in this initial implementation.